api/src/utils/seminuevosHelperFunctions.ts

```diff
@@ -0,0 +1,95 @@
+import { Page } from 'puppeteer'
+import { wait } from '@utils/helperFunctions'
+import path from 'path'
+import { Ad, Dropdown } from '@customTypes/index'
+
+export const waitForResponse = async ({
```

Comment:
Adding this function for the cases when we need to wait for a browser response before moving to the next step. For example, the dropdowns where the options change based on the selection of a previous one